recipes/pybufr-ecmwf/meta.yaml

```diff
@@ -0,0 +1,63 @@
+# Note: there are many handy hints in commenets in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set version = "0.81" %}
+{% set sha256 = "abe0f47bfcac9603c4b47a9dc7d95befed6ae3091ef87f2511e42bbfa5b52e35" %}
+{% set sha2563k = "b4027e018188a1ea29ab3fb5374087bac569c290335a1d5cde8356b446069435" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#   $ openssl sha256 <file>
+# On Windows, you'll need the openssl pacakge, available on conda-forge
+
+package:
+  name: pybufr-ecmwf  # [py2k]
+  name: pybufr-ecmwf3  # [py3k]
+  version: {{ version }}
+
+source:
+  fn: pybufr-ecmwf-{{ version }}.tar.gz  # [py2k]
+  fn: pybufr-ecmwf3-{{ version }}.tar.gz  # [py3k]
+  url: https://pypi.io/packages/source/p/pybufr-ecmwf/pybufr-ecmwf-{{ version }}.tar.gz  # [py2k]
+  url: https://pypi.io/packages/source/p/pybufr-ecmwf3/pybufr-ecmwf3-{{ version }}.tar.gz  # [py3k]
+  sha256: {{ sha256 }}  # [py2k]
+  sha256: {{ sha2563k }}  # [py3k]
+
+build:
+  skip: true  # [win]
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  script: python setup.py install
+
+requirements:
+  build:
+    - numpy x.x
+    - python
+    - gcc
+    - libgfortran 
```

Comment:
`libgfortran` is in the `gcc` packag, so you can remove it from here. You will need it as a run time dependency (below).